src/examples/dropdowns/combobox/Default.tsx

```diff
@@ -0,0 +1,91 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React, { useMemo, useState, useEffect } from 'react';
+import debounce from 'lodash.debounce';
+import { filter as fuzzyFilter } from 'fuzzyjs';
+import { Item, Menu, Label, Field, Dropdown, Combobox } from '@zendeskgarden/react-dropdowns';
+import { Row, Col } from '@zendeskgarden/react-grid';
+import { ReactComponent as SearchIcon } from '@zendeskgarden/svg-icons/src/16/search-stroke.svg';
+
+const options = [
+  'Asparagus',
+  'Brussel sprouts',
+  'Cauliflower',
+  'Garlic',
+  'Jerusalem artichoke',
+  'Kale',
+  'Lettuce',
+  'Onion',
+  'Mushroom',
+  'Potato',
+  'Radish',
+  'Spinach',
+  'Tomato',
+  'Yam',
+  'Zucchini'
+];
+
+const Example = () => {
+  const [selectedItem, setSelectedItem] = useState('');
+  const [inputValue, setInputValue] = useState('');
+  const [matchingOptions, setMatchingOptions] = useState(options);
+
+  /**
+   * Debounce filtering
+   */
+  const filterMatchingOptions = useMemo(
+    () =>
+      debounce(
+        (value: string) => {
+          const selectedOptions = options.filter(fuzzyFilter(value, { iterator: item => item }));
+
+          setMatchingOptions(selectedOptions);
+        },
+        300,
+        { leading: true }
+      ),
+    [setMatchingOptions]
+  );
+
+  useEffect(() => {
+    filterMatchingOptions(inputValue);
+  }, [filterMatchingOptions, inputValue]);
+
+  return (
+    <Row justifyContent="center">
+      <Col sm={5}>
+        <Dropdown
+          inputValue={inputValue}
+          selectedItem={selectedItem}
+          onSelect={item => setSelectedItem(item)}
+          onInputValueChange={value => setInputValue(value)}
+        >
+          <Field>
+            <Label>Choose a vegetable</Label>
+            <Combobox start={<SearchIcon />} placeholder="Search vegetables">
+              {selectedItem}
+            </Combobox>
+          </Field>
+          <Menu>
+            {matchingOptions.length ? (
+              matchingOptions.map(option => (
+                <Item key={option} value={option}>
+                  <span>{option}</span>
+                </Item>
+              ))
+            ) : (
+              <Item disabled>No matches found</Item>
+            )}
+          </Menu>
+        </Dropdown>
+      </Col>
+    </Row>
+  );
+};
+
+export default Example;
```

Comment:
I noticed that when a user removes the characters from the text box after a search, the list box with all the options flash for a split second. If this is a bug, is there a way to fix it?

![2021-12-08 09 33 19](https://user-images.githubusercontent.com/1811365/145255887-309e502f-fcc9-4da9-b918-870e641c337d.gif)

Reply:
It's like it needs an extra ms of `debounce` maybe 🤔

Reply:
I think I introduced this behavior while adding the option `{ leading: true }` to the `debounce` function. I removed it and added better indication of when the Combobox is searching.